.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: rust
+os:
+  - linux
+  - osx
+rust:
+  - stable
+  - nightly
+matrix:
+  allow_failures:
+    - rust: nightly
+notifications:
+  email: false
+before_script:
+  - export PATH=$PATH:~/.cargo/bin
+script:
+  - cargo build --verbose
+  - cargo test --verbose
+after_success:
```

Comment:
Could we remove this section? I'd prefer to just let crates.io handle the documentation (of which there is very little currently 😅 ). It also references rsx_parser currently and is passing a github token to a script I don't necessarily trust. I'd rather just avoid any issues here and let crates.io take care of the docs.

Reply:
Sure thing. I was thinking you'd change these yourself, along with the readme badge, but I'll remove these.

Reply:
Ah gotcha, yeah I'll change the badge and omit the `after_success` part of CI. Thanks!